Not being able to run the server and client

-Node version: 4.1.2

-This is the error log:
Error: Cannot find module 'clear'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at /Users/hellacio/Desktop/plushie-badname/server.js:11:21
    at Object.<anonymous> (/Users/hellacio/Desktop/plushie-badname/server.js:35:7)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)

--- a/package.json
+++ b/package.json
@@ -25,5 +25,8 @@
     "url": "https://github.com/temmihoo/plushie-badname/issues"
   }
 , "homepage": "https://github.com/temmihoo/plushie-badname#readme"
-, "dependencies": { "coap": "0.12.0" }
+, "dependencies": {
+  "coap": "0.12.0"
+, "clear": "0.0.1"
+}
 }

--- a/package.json
+++ b/package.json
@@ -25,4 +25,5 @@
     "url": "https://github.com/temmihoo/plushie-badname/issues"
   }
 , "homepage": "https://github.com/temmihoo/plushie-badname#readme"
+, "dependencies": { "coap": "0.12.0" }
 }

